app/assets/javascripts/application.js

```diff
@@ -14,4 +14,4 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
-//= require bootstrap-sprockets
\ No newline at end of file
+= require bootstrap-sprockets
```

Comment:
I don't think this change is actually needed.

It's absolutely absurd, I know, but `//=` is a directive, not a commented out line, and I think removing the `//` may have some unwanted effects